docs/guide/targetgroupbinding/targetgroupbinding.md

```diff
@@ -16,8 +16,25 @@ TargetGroupBinding CR supports TargetGroups of either `instance` or `ip` TargetT
 !!!tip ""
     If TargetType is not explicitly specified, a mutating webhook will automatically call AWS API to find the TargetType for your TargetGroup and set it to correct value.
 
+## Choosing the Target Group
+One can either use ``targetGroupARN`` of ``targetGroupName`` to identify a Target Group. Although both are unique and immutable in an AWS region, one only has control of the ``targetGroupName``, for ``targetGroupARN`` is generated by AWS and contain random characters.
+
+If you provide both ``targetGroupARN`` and ``targetGroupName``, beware that ``targetGroupARN`` prevails.
+
+
+## Sample YAMLs
+```yaml
+apiVersion: elbv2.k8s.aws/v1beta1
+kind: TargetGroupBinding
+metadata:
+  name: my-tgb
+spec:
+  serviceRef:
+    name: awesome-service # route traffic to the awesome-service
+    port: 80
+  targetGroupName: <name-of-the-targetGroup>
+```
 
-## Sample YAML
```

Comment:
Was this deletion intended?

Reply:
Hi! This line wasn't really deleted. It changed to `## Sample YAMLs` (plural) and moved about 10 lines above :)